test/end-to-end.js

```diff
@@ -29,8 +29,15 @@ var MYSQL_DATABASE = 'loopback_workspace_test';
 var MYSQL_USER = 'lbws';
 var MYSQL_PASSWORD = 'hbx42rec';
 
+before(function cleanTestDir(cb) {
+  fs.remove(PKG_CACHE, function(err) {
```

Comment:
Ouch, this completely cancels the point of using `strong-cached-install`. If we are going to delete `PKG_CACHE` before the tests run, then it's probably better to use `npm install` directly without `strong-cached-install`.

What problem is this trying to solve? Also how can be `PKG_CACHE` non-empty on CI machines, when each CI run starts in a clean cloned/checked-out repository?

I introduced `strong-cached-install` to speed up `npm test` when it's run repeatedly on a developer's machine. I.e. the first `npm test` is pretty slow, but subsequent `npm test` runs are much faster, as `npm install` is replaced with recursive copy.

Can we revert this change?